athena-aws-cmdb/src/main/java/com/amazonaws/athena/connectors/aws/cmdb/TableProviderFactory.java

```diff
@@ -59,14 +59,14 @@ public TableProviderFactory(java.util.Map<String, String> configOptions)
     {
         this(
             AmazonEC2ClientBuilder.standard().build(),
-            AmazonElasticMapReduceClientBuilder.standard().build(),
+            EmrClient.builder().region(DefaultAwsRegionProviderChain.builder().build().getRegion()).build(),
```

Comment:
Use `EmrClient.create()`. It uses the default aws region provider chain.

Reply:
updated with the changes suggested.